patterns/homepage.php

```diff
@@ -0,0 +1,230 @@
+<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}}},"backgroundColor":"background","textColor":"primary","className":"course-home-page-responsive-padding","layout":{"type":"default","contentSize":"1200px"}} -->
+<div class="wp-block-group alignfull course-home-page-responsive-padding has-primary-color has-background-background-color has-text-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:spacer {"height":"10px"} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"primary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-primary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-gothic-font-family"} -->
+<h3 class="has-gothic-font-family" style="text-transform:uppercase">pages</h3>
+<!-- /wp:heading -->
+
+<!-- wp:group {"className":"has-eb-garamond-font-family","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group has-eb-garamond-font-family"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size">About</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Video</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Episodes</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>All Posts</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"primary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-primary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-gothic-font-family"} -->
+<h3 class="has-gothic-font-family" style="text-transform:uppercase">social</h3>
+<!-- /wp:heading -->
+
+<!-- wp:group {"className":"has-eb-garamond-font-family","layout":{"type":"flex","flexWrap":"nowrap"}} -->
+<div class="wp-block-group has-eb-garamond-font-family"><!-- wp:paragraph {"fontSize":"small"} -->
+<p class="has-small-font-size">Instagram</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Youtube</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Spotify</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group -->
+
+<!-- wp:heading {"textAlign":"center","className":"has-sorts-mill-goudy-font-family","fontSize":"5x-large"} -->
+<h2 class="has-text-align-center has-sorts-mill-goudy-font-family has-5-x-large-font-size">COURSE</h2>
```

Comment:
I think we should move the footer to the template part instead of keeping it here.